### PR TITLE
Data selfie

### PR DESCRIPTION
**Pull request by beaugunderson**

### CODE REVIEW

File: open_humans/templates/member/my-member-data-selfie.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+</div>
+{% endblock %}
+
+{% block dashboard_main %}
+<div class="panel panel-default pad-all-sides">
+  {% with source='data_selfie' %}
+    {% source_is_public source as is_public %}
+    {% source_is_connected source user as is_connected %}
+
+    {% include 'partials/share-and-delete-buttons.html' %}
```
Comment (madprime) [inline open_humans/templates/member/my-member-data-selfie.html]: The public sharing button, as included on this page, directs back to research data when someone clicks "share publicly". I think the user should "stay" (or actually stay, with AJAX?) on this page when they update public status on this button.

**Comment by madprime** [comment]
Just had one bump in user experience I commented on, otherwise :+1: I look forward to seeing this is staging!

File: open_humans/templates/member/my-member-data-selfie.html (hunk 1 above)
Comment (beaugunderson) [inline open_humans/templates/member/my-member-data-selfie.html]: ah yes, the redirect there is unintentional, it should be using AJAX—I’ll investigate

Comment (beaugunderson) [inline open_humans/templates/member/my-member-data-selfie.html]: oh, needs the JavaScript included on that page, simple fix :)

**Pull request merged**